frontend/src/pages/cart/Cart.jsx

```diff
@@ -113,9 +113,7 @@ export const Cart = () => {
       const storedUserId = localStorage.getItem('userID');
       if (storedUserId) {
         navigate(`/cart/${storedUserId}`);
-      } else {
-        // Handle case where there is no user ID (e.g., prompt login or show error)
-      }
+      } 
     }
   }, [userId, navigate]);
   
```

Comment:
I removed this part from the Cart.jsx as no routes lead to this page without being logged in first thing (changes made on cart button on  landing page to /login, thus accessed only via /profile/:id after login step)